.travis.yml

```diff
@@ -4,8 +4,23 @@ go:
   - 1.8
 
 dist: trusty
-sudo: false
+sudo: required
```

Comment:
The only viable alternative I've found: Instead of running `docker-compose build` on travis (and therefore needing sudo to run the daemon), we could get rid of docker-compose, run only `docker build` instead and do this in gcloud instead of travis with `gcloud docker -- build <image-to-be-build>`. Se: https://cloud.google.com/sdk/gcloud/reference/docker
This could be faster, too.